File: Beecrowd/Beginner/1101 - 1190/1181 - Line in Array/C/1181_LineinArray.c
#include <stdio.h>

int main() {
    int line;
    char operation;
    double sum = 0.0, number;
    
    scanf("%d %c", &line, &operation);

    for (int i = 0; i < 12; i++) {
        for (int j = 0; j < 12; j++) {
            scanf("%lf", &number);
            
            if (i == line) {
                sum += number;
            }
        }
    }

    if (operation == 'S') printf("%.1lf\n", sum);
    else printf("%.1lf\n", sum / 12.0);

    return 0;
}

File: Beecrowd/Beginner/1101 - 1190/1181 - Line in Array/Python/1181_LineinArray.py
line = input()
operation = input()
line = int(line)
sum_val = 0.0

for i in range(12):
    for j in range(12):
        number = float(input())

        if i == line:
            sum_val += number

if operation == 'S':
    print(f"{sum_val:.1f}")
else:
    print(f"{sum_val / 12.0:.1f}")
